source/appendices/a-glossary-of-terms.rst

```diff
@@ -459,9 +459,7 @@ Note: The main Code of Practice document takes precedence in the case of any con
      -
 
    * - Full_Content_Database
-     - A COUNTER Host_Type for report providers that offer databases that are a collection of content items that are not otherwise part of a serial or monograph (i.e. non-aggregated).
-
-       Note: In contrast to A&I_Databases and Aggregated_Full_Content the Investigations and Requests for Full_Content_Databases (like for example Cochrane Database of Systematic Reviews) are reported with Data_Type Database.
+     - A COUNTER Host_Type for report providers that offer databases that are a collection of content items that are not aggregated into titles (i.e. not part of a serial or book or other title). Full_Content_Database may include but not be exclusively composed of multimedia content items.
```

Comment:
While reading the multimedia comment (which differs from the one above - this one seems better to me) I first thought that this would be wrong, but I've mixed Host_Types and Data_Types. A database with multimedia content only would indeed result in Host_Type Multimedia_Collection, but the Searches and Denial metrics would be reported with Data_Type Database_Full in DR.